packages/asc-ui/src/components/Pagination/Pagination.tsx

```diff
@@ -62,6 +62,11 @@ const Pagination: FunctionComponent<
     }
   }, [paginationLength])
 
+  // If page prop changes, set currentPage so component can be managed externally
+  useEffect(() => {
+    setCurrentPage(page)
```

Comment:
Nice. Can you also add a test for this addition?

Reply:
Yesss, done!